packages/@react-aria/tag/test/useTag.test.js

```diff
@@ -13,7 +13,7 @@
 import {renderHook} from '@testing-library/react-hooks';
 import {useTag} from '../';
 
-describe('useTag tests', () => {
+describe.skip('useTag tests', () => {
```

Comment:
what is breaking that we need to skip all of these?

Reply:
I accidentally left this in. But since we're on the subjects of the useTag test, I've been having trouble mocking an items collection to get a feasible test environment. Do you have an advice?

Reply:
We probably don't need a useTag test, they'll only exist inside a TagGroup right? so as long as we have good integration level tests, we should be fine